pilot/pkg/networking/core/v1alpha3/sidecar_simulation_test.go

```diff
@@ -2445,7 +2445,47 @@ spec:
 					for _, tc := range tt.cfg {
 						cfg = cfg + "\n---\n" + tc.Config(variant)
 					}
-					s := xds.NewFakeDiscoveryServer(t, xds.FakeOptions{ConfigString: cfg})
+					s := xds.NewFakeDiscoveryServer(t, xds.FakeOptions{
+						ConfigString: cfg,
+						Services: []*model.Service{
+							{
+								Hostname: "alt-known-default.example.com",
```

Comment:
This is pretty annoying to have to do everywhere.

I think in general there are two different cases here -- reference _not allowed_ and reference unknown. I wonder if we can have unknown reference generate the same VirtualService but just report an issue in status? 

For not allowed we should explicitly deny it probably though